grammars/fixtures/fenced-code.cson

```diff
@@ -73,6 +73,7 @@ list: [
   { pattern:'stata' }
   { pattern:'swift' }
   { pattern:'xml', include:'text.xml', contentName:'text.embedded.xml' }
+  { pattern:'yaml' }
 
   # NOTE: temporarily disabled, see issue #77 for details
```

Comment:
Have you verified that this doesn't cause the same issues as noted in #77?

Reply:
I hadn't seen that issue. In retrospect there must've been a reason for it to be missing. I've tested it with several YAML examples but can't seem to find a bug yet. I thought I saw you mention somewhere that unquoted strings in YAML blocks messed up the rest of the document, but that doesn't happen for me.

Reply:
Ok, thanks. It's not fresh in my memory either, so I'll have to give it a test when some time frees up. Thanks for the PR in any case.